### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/github-backup-utils/commit/ac5b614ad858465bc65c9e8b980a3fdb8a9e799b#commitcomment-74625966

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/github-backup-utils/commit/cdd0c85e458fe650c2db7b2240659dca345f1b34

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ For example, Backup Utilities 2.13.0 can be used to backup and restore all patch
 
 There is no need to align Backup Utilities patch releases with GitHub Enterprise Server patch releases.
 
-When making a `.0` release, you will need to specify the minimum supported version of GitHub Enterprise Server that that release supports.
+When making a `.0` release, you will need to specify the minimum supported version of GitHub Enterprise Server that release supports.
```
Comment (jsoref) [inline RELEASING.md:9]: This is arguable. I think both forms are ok. Happy to drop.

File: share/github-backup-utils/ghe-backup-config [hunk 2]
```diff
@@ -335,7 +335,7 @@ ssh_host_part() {
 # rsync commands.
 ssh_port_part() {
   if [ "${1##*:}" != "$1" ] && [ "${1##*:}" -ne "122" ]; then
-    echo "Error: SSH port has to be 122 connecting to Github Enterprise Server, current value is ${1##*:} for $1." 1>&2
+    echo "Error: SSH port has to be 122 connecting to GitHub Enterprise Server, current value is ${1##*:} for $1." 1>&2
```
Comment (jsoref) [inline share/github-backup-utils/ghe-backup-config:338]: Brand

**Comment by jsoref** [comment]
Test failures appear to be @bonsohi https://github.com/github/backup-utils/pull/913#issuecomment-1139016893

**Comment by jsoref** [comment]
@elee do I need to do something to push this along?

**Comment by elee** [comment]
@jsoref looks good to me

**Review by elee — APPROVED** [review]

**Pull request merged**